src/test/Components/ImagesTable/ImagesTable.test.tsx

```diff
@@ -53,14 +53,15 @@ describe('Images Table', () => {
     const { getAllByRole } = within(table);
     const rows = getAllByRole('row');
     // remove first row from list since it is just header labels
-    const header = rows.shift();
+    const header = rows.shift() as HTMLElement;
```

Comment:
Instead of using `as HTMLElement`, can we use `const header: HTMLElement = rows.shift()`?

`as` forces the Typescript linter to accept it as true, but the way I propose above allows the linter to do type checking as we want.

Reply:
Done...